pkg/asset/agent/image/ignition.go

```diff
@@ -75,6 +75,7 @@ type agentTemplateData struct {
 	ConfigImageFiles          string
 	ImageTypeISO              string
 	PublicKeyPEM              string
+	Token                     string
```

Comment:
Nit: Do you want the name to be more specific like `AuthToken`? `Token` seems a bit generic.